README.md

```diff
@@ -1 +1,45 @@
-﻿# Todo List App
+
+# AwesomeBooks
+
+This is a wesbite where you can add and remove books.
+
+![screenshot](Screenshottt.png)
+
+This project is my portfolio.
```

Comment:
- [ ] [OPTIONAL] It is ok if you use the readme file from an other project, just make sure to delete all the information from the previous project as it makes the project look unorganized and less professional.